autopilot/contracts.go

```diff
@@ -1,27 +1,413 @@
 package autopilot
 
 import (
+	"math"
+	"time"
+
+	"gitlab.com/NebulousLabs/errors"
+	"go.sia.tech/renterd/bus"
 	"go.sia.tech/renterd/internal/consensus"
+	rhpv2 "go.sia.tech/renterd/rhp/v2"
 	"go.sia.tech/renterd/worker"
 	"go.sia.tech/siad/types"
 	"golang.org/x/crypto/blake2b"
 )
 
-func (ap *Autopilot) currentHeight() uint64 {
-	return ap.store.Tip().Height
+const (
+	// contractSetName defines the name of the default contract set
+	contractSetName = "autopilot"
+
+	// contractLoopInterval defines the interval with which we run the contract loop
+	contractLoopInterval = 15 * time.Minute
+
+	// estimatedFileContractTransactionSetSize is the estimated blockchain size
+	// of a transaction set between a renter and a host that contains a file
+	// contract.
+	estimatedFileContractTransactionSetSize = 2048
```

Comment:
(nit) I think we should put these estimates into a single place. Because we also got some for the wallet.